src/index.ts

```diff
@@ -35,9 +37,16 @@ const createJWKSMock = (
     stop,
     kid,
     token,
+    jwksHandler,
   }
 }
 
-export type JWKSMock = ReturnType<typeof createJWKSMock>
+export type JWKSMock = {
+  start: () => void
+  stop: () => void
+  kid: () => string
+  token: (token: JwtPayload) => string
+  jwksHandler: (options?: RequestHandlerOptions) => HttpHandler
```

Comment:
```suggestion
  createMSWRequestHandler: (options?: RequestHandlerOptions) => HttpHandler
```
I think it should be somehow clear from the name that this is for `msw`. Will implement respective changes.